runtime/oti/ContinuationHelpers.hpp

```diff
@@ -370,8 +370,13 @@ class VM_ContinuationHelpers {
 			} else {
 				/* Set the notified and onWaitingList flags for virtual threads that have not been interrupted. */
 				J9VMJAVALANGVIRTUALTHREAD_SET_NOTIFIED(vmThread, current->vthread, JNI_TRUE);
-				/* Update the thread state to BLOCKED. */
-				J9VMJAVALANGVIRTUALTHREAD_SET_STATE(vmThread, current->vthread, JAVA_LANG_VIRTUALTHREAD_BLOCKED);
+
+				U_32 state = J9VMJAVALANGVIRTUALTHREAD_STATE(vmThread, current->vthread);
```

Comment:
`current->vthread` is being accessed multiple times ... it should be stored in local variable